weave-js/src/components/Button/Button.tsx

```diff
@@ -32,6 +33,7 @@ export type ButtonProps = React.ButtonHTMLAttributes<HTMLButtonElement> & {
   children?: ReactElement | string;
   active?: boolean;
   tooltip?: string;
+  tooltipProps?: TooltipContentProps;
```

Comment:
previously picked out `align` and `side` props specifically, but realized we might want full flexibility including `alignOffset`, `sideOffset`, and even `className` so I made this prop broader